src/table/Root.tsx

```diff
@@ -50,8 +51,9 @@ export function renderPaginationTable(props: RenderProps, reactRoot?: ReactDom.R
           changeSortOrder={changeSortOrder}
           applyColumnWidths={applyColumnWidths}
           rect={rect}
+          viewService={viewService}
         >
-          <TableWrapper {...wrapperProps} direction={direction} />
+          <TableWrapper {...wrapperProps} direction={direction} viewService={viewService} />
```

Comment:
`viewService` is passed to the context provider and the `TableWrapper` component?

Reply:
Updated!